MODULE.bazel

```diff
@@ -40,6 +40,7 @@ bazel_dep(name = "boost.property_tree", version = BOOST_VERSION + ".bcr.2")
 bazel_dep(name = "boost.spirit", version = BOOST_VERSION)
 bazel_dep(name = "boost.stacktrace", version = BOOST_VERSION)
 bazel_dep(name = "boost.thread", version = BOOST_VERSION)
+bazel_dep(name = "boost.filesystem", version = BOOST_VERSION)
```

Comment:
.. add ".bcr.1" to BOOST_VERSION instead. I think that version has the glob fixed.

```
bazel_dep(name = "boost.filesystem", version = BOOST_VERSION + ".bcr.1")
```

Reply:
will try...